### PR TITLE
[CDAP-12692] Remove deprecated schedule codes

### PR DESCRIPTION
**Pull request by ecfm**
PR is organized in separated commits

On-hold until we decide what to do with StreamSizeSchedule

### CODE REVIEW

**Comment by gsps1** [comment]
`RunConstraints` have to be removed too.

File: cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java [hunk 1]
```diff
@@ -674,23 +659,17 @@ public void getProgramStatusSchedules(HttpRequest request, HttpResponder respond
...
+    responder.sendJson(HttpResponseStatus.OK, GSON.toJson(details, Schedulers.SCHEDULE_DETAILS_TYPE));
   }
 
   @GET
   @Path("apps/{app-name}/schedules/{schedule-name}")
   public void getSchedule(HttpRequest request, HttpResponder responder,
                           @PathParam("namespace-id") String namespaceId,
                           @PathParam("app-name") String appName,
-                          @PathParam("schedule-name") String scheduleName,
-                          @QueryParam("format") String format)
```
Comment (gsps1) [inline cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/ProgramLifecycleHttpHandler.java:694]: these will need docs change. you can update that in a separate PR if needed.

File: cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/TimeSchedulerService.java [hunk 2]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright © 2014-2017 Cask Data, Inc.
+ * Copyright © 2014 Cask Data, Inc.
```
Comment (gsps1) [inline cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/TimeSchedulerService.java:2]: undo.

**Comment by gsps1** [comment]
build green : https://builds.cask.co/browse/CDAP-DUT6244-1

**Comment by anwar6953** [comment]
I think you should add to the JIRA's release notes the functionality that has been removed (such as stream-size-based schedules).

File: cdap-app-fabric/src/test/java/co/cask/cdap/AppWithStream.java [hunk 3]
```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright © 2015-2017 Cask Data, Inc.
```
Comment (anwar6953) [inline cdap-app-fabric/src/test/java/co/cask/cdap/AppWithStream.java]: nit - this can just be `2017` right?

**Review by anwar6953 — APPROVED** [review]
One comment about the release notes; remainder LGTM.

**Pull request merged**